src/patterns/OcSearchBar.vue

```diff
@@ -22,21 +27,30 @@
         />
       </div>
     </div>
-    <div v-if="button" class="uk-width-auto">
+    <div :class="[{ 'oc-visually-hidden': buttonHidden }, 'uk-width-auto']">
       <oc-button
         class="oc-search-button"
         :class="{ 'uk-button-small': small }"
         variation="primary"
         :disabled="loading || searchQuery.length < 1"
         @click="onSearch"
-        >{{ button }}</oc-button
+        >{{ buttonLabel }}</oc-button
       >
     </div>
   </div>
 </template>
 <script>
 /**
  * The search bar is an input element used for searching server side resources or to filter local results.
+ *
+ * ## Accessibility
+ *
+ * ### Landmark role=search
+ * Given there is only one instance of `<oc-search-bar>` per page/route, this component should communicate its purpose, being the main site search, to screen readers ([explainer of landmark roles](https://www.washington.edu/accessibility/web/landmarks/)). If the component serves as a filter form, it is advised to disable the landmark role via `isFilter="true"`.
```

Comment:
What about the case when there would be two search bars but none of them as a filter? Let's say we have a global search and then some search bar that would be only app-related? Should then both have the "search" role?

Reply:
Technically speaking: if there are two landmark roles of the same kind you are advised to label them in order to make them differentiable.

Speaking in terms of best practices regarding landmarks: `role=search` should only mark **the** main search form for an interface (global in this case). It's kind of hard to theorize it even further, but what's the global search should be consistent across routes/states/contexts ([WCAG 3.2.4 consistent identification](https://www.w3.org/TR/WCAG21/#consistent-identification)).

Reply:
Thanks for explanation!